package.json

```diff
@@ -258,7 +258,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "6.9.2",
+    "@department-of-veterans-affairs/formation": "6.9.3",
```

Comment:
I would prefer to see dependency upgrades as separate PRs.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/13405

Reply:
pulled in #13405 to this branch